source/_integrations/airq.markdown

```diff
@@ -21,7 +21,7 @@ The **air-Q** {% term integration %} allows integrating the sensors, provided by
 
 {% include integrations/config_flow.md %}
 
-During the configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.
+During the initial configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.
```

Comment:
```suggestion
```
I realize this sentence was already in this position before this PR. But I think it makes sense to put it before the config flow, as a precondition. This is also done in other integrations where credentials are required.